plutus-playground/default.nix

```diff
@@ -0,0 +1,9 @@
+{
+  nixpkgs ? builtins.fetchTarball "https://releases.nixos.org/nixos/18.09/nixos-18.09.1023.06fb0253afa/nixexprs.tar.xz"
```

Comment:
please import lib.nix and use pkgs from there so the same rev is used across the entire project.

Reply:
will do

Reply:
Please fix.

Reply:
In fact it seems like this whole file should not exist and instead go via the main `default.nix`.